content/en/blogs/TiDB-on-KubeSphere-using-qke.md

```diff
@@ -28,15 +30,15 @@ Therefore, I select QingCloud Kubernetes Engine (QKE) to prepare the environment
 
 1. Log in the [web console of QingCloud](https://console.qingcloud.com/). Simply select KubeSphere (QKE) from the menu and create a Kubernetes cluster with KubeSphere installed. The platform allows you to install different components of KubeSphere. Here, we need to enable [OpenPitrix](https://github.com/openpitrix/openpitrix), which powers the app management feature in KubeSphere.
 
-![qingcloud-kubernetes-engine](https://ap3.qingstor.com/kubesphere-website/docs/20201026173924.png)
+> Note: KubeSphere can be installed on any infrastructure, we just use the QingCloud platform as an example, see [KubeSphere Documentation](https://kubesphere.io/docs/) for more details.
```

Comment:
->
{{< notice note >}}
KubeSphere can be installed on any infrastructure. I just use QingCloud Platform as an example. See [KubeSphere Documentation](https://kubesphere.io/docs/) for more details.
{{</ notice >}}